contracts/src/token/erc721/mod.rs

```diff
@@ -193,8 +198,9 @@ unsafe impl TopLevelStorage for Erc721 {}
 
 /// Required interface of an [`Erc721`] compliant contract.
 pub trait IErc721 {
-    /// The error type associated to this ERC-721 trait implementation.
+    /// The error type associated with ERC-721 contract.
```

Comment:
I think what we had here before was better. Same with the other comments.

P.s. this should be `with the ERC-721 contract` if we keep it.